lib/resources/r4/allergy_intolerance.yaml

```diff
@@ -62,21 +62,7 @@ fields:
   cardinality: 0..*
   type: code
   description: low | high | unable-to-assess
-  url: http://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.criticality
-  binding:
-    description: Assertion about certainty associated with the propensity, or potential risk, of a reaction to the
-                 identified substance (including pharmaceutical product).
-    terminology:
-    - display: AllergyIntoleranceCriticality
-      system: http://hl7.org/fhir/allergy-intolerance-criticality
-      info_link: http://hl7.org/fhir/r4/valueset-allergy-intolerance-criticality.html
-
```

Comment:
Are we missing 'Encounter' and 'Patient' fields?

Reply:
for terminology bindings?

Reply:
@praneethk2094 @sbanda51 this thread doesn't have a resolution

Reply:
Discussed offline, we do not require Encounter and patient fields for terminology bindings.